applications/DAOsign.md

```diff
@@ -0,0 +1,186 @@
+
+
+Deliveries include DAOsign platform with an ability to:
+- create agreements including generating using ChatGPT
+- add and remove signers and observers of different levels of anonymity
+- allow signing the documents with further storing of signature proofs on Astar testnet blockchain.
+
+
+**Core DAOsign components:**
+
+<img width="902" alt="Screenshot 2023-05-27 at 16 51 43" src="https://github.com/Burmenska/Grants-Program/assets/3983591/e6e3e78d-edf4-4d50-a298-e01210d75963">
+
+
+**Technical design:**
+
+Proofs are implemented in the EIP-712 standard and adjusted to ink! ecosystem. Proofs digned using user’s EVM private key. 
+We are supporting this standard in Ink! smart contract to be able to verify the signature in ink! smart contract. 
+Also, we are implementing an adjusted EIP-2771 standard to allow Agreement Signers gas-free delegation to publish proof on-chain.
+EIP-712 standard will be used for proofs, adaptation means that user data is converted to a binary bit; we use its hash (serialization vs deserialization)
+And EIP-2771 standard - to securely send the proof and receive it.
+So that EVM standards will be adapted to ink! ecosystem requirements. 
+Example (interfaces) of the smart contracts can be found here https://github.com/DAOsign/daosign-smart-contracts  (Solidity version).
+
+
+Detailed storyboard can be found here: https://consideritdone.storiesonboard.com/m/PUCLDvQC4kKwoRJ8kbm3MA .
+
+Architectural diagram:
+
+<img width="805" alt="Screenshot 2023-05-28 at 23 30 53" src="https://github.com/Burmenska/Grants-Program/assets/3983591/54c8708d-e180-46f3-9a78-b7121de133d3">
+
+### Ecosystem Fit
+
+DAOsign fit the Governance category and since, Polkadot and Kusama are designed to be community-driven networks that allow for decentralized governance. DAOsign can fit into this ecosystem by providing a platform for DAOs to manage their decision-making processes in a decentralized manner.
+Our target audience is not only DAOs, but also all the companies, who have a need to deal with the agreement/policies management including the use case for managing Grants, that was presented to you during our introdution call.
+Competitors analysis can be found here: https://docsend.com/view/sbexkxn5su3ahdg8 . Particularly in Substrate / Polkadot / Kusama ecosystems we are not aware about any tools, that provide exactly the same functionality. 
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Team Leader, Founder: Eugene Fine https://github.com/ConsiderItDone,
+- Solution Architect: Ramil Amerzyanov https://github.com/ramilexe
+- Blockchain / Tech lead: Misha Kushka https://github.com/kushkamisha
+- Director of Buisiness development and Partnerships: Oleksandra Burmenska https://github.com/burmenska
+
+
+
+### Contact
+
+- **Contact Name:** Eugene Fine
+- **Contact Email:** eugene@daosign.org
+- **Website:** daosign.org
+
+### Legal Structure
+
+- **Registered Address:** ILLINOIS LIMITED LIABILITY CORPORATION, US
```

Comment:
This seems to be a company rather than an address.

Reply:
Updated in the doc